testing_and_setup/compass/ocean/periodic_planar/2.5km/default_gotm/config_init.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<config case="init">
+
+	<add_executable source="model" dest="ocean_model"/>
+
+	<namelist name="namelist.ocean" mode="init">
+		<option name="config_init_configuration">'periodic_planar'</option>
+		<option name="config_vert_levels">-1</option>
+		<option name="config_periodic_planar_vert_levels">25</option>
```

Comment:
Should be `250` to match the test in section 5.1 of [Kärnä, 2020](https://doi.org/10.1016/j.ocemod.2020.101619)